hash/dc.xml

```diff
@@ -1059,7 +1059,7 @@ P830-****-** : Appears to be a Sega part number for promo discs.
 		</part>
 	</software>
 
-	<!-- This is labeled "Rev. A". It is unknown if there ever was a "Rev. 0" released. -->
+	<!-- This is labeled "RevRev. A". It is unknown if there ever was a "Rev. 0" released. -->
```

Comment:
This looks like a typo. This reduplication "RevRev" is correct?

Reply:
Correction made. Thanks!